website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,407 @@
+---
+id:  implementation-of-flint-modules
+title:  Implementation of FLINT Modules
+slug:  /implementation-flint-modules
+description:  Know and understand the FLINT Modules and how they work, along with the contribution of each FLINT Module
+image: https://i.imgur.com/An53MDO.png
+keywords:
+- FLINT Modules
+- FLINT
+- Modules
+- GCBM
+- Hybrid Forest
+- Empirical Growth
+- Turnover module
+- Decomposition module
+- Chapman Richards
+- Carbon cycle
+- moja global
+- WOFOST crop growth
+---
+
+![Cover Image for the case study titled `Implementation of FLINT Modules`](../static/img/implementation-of-flint-modules.png)
+
+FLINT is a modular tool that supports better land management and helps reduce greenhouse gas (GHG) emissions worldwide, and allows multiple models and data required to better understand land management to be brought together in one place. 
+
+A module is a self-contained set of operations that determine the state of, or change in, variables across a specified period of time for a single Simulation Unit in direct response to event notifications from the FLINT core system (Unit Controller). For example, the empirical forest growth module includes all the operations required to simulate biomass accumulation.
+
+Modules are all written to work with the [`develop`](https://github.com/moja-global/FLINT/tree/develop) branch of the **FLINT** and any actively maintained branches will work with the most recent version (and are broken with old enough builds of the FLINT). The FLINT Application Programming Interface (API) has evolved over time, so some modules might be out of date. None of these modules require a custom fork of FLINT.
+
+## Background
+
+It is useful to think about what a module manifest might look like. **Manifests** are meta-data used by the package managers to describe the compatibility (FLINT versions that are used), dependencies (necessary inputs) and the configuration requirements (for example whether the module uses daily, monthly or annual time steps).
```

Comment:
This doesn't really provide any background material to modules. I think it can be merged into 'How do modules work'. Remove the mention of manifests, I don't think it helps to introduce new concepts yet.